### PR TITLE
[TE] frontend - ttbach/add session update

### PR DESCRIPTION
**Pull request by ttbach**
Added information about last user and time anomaly was updated

screen shot 2018-01-18 at 5 16 01 pm

### CODE REVIEW

File: thirdeye/thirdeye-frontend/app/pods/rootcause/route.js [hunk 1]
```diff
@@ -231,6 +233,7 @@ export default Ember.Route.extend(AuthenticatedRouteMixin, {
     // session-initialized context
     if (sessionId) {
       if (!_.isEmpty(session)) {
+        const { name, text, updatedBy, updated } = model.session;
```
Comment (apucher) [inline thirdeye/thirdeye-frontend/app/pods/rootcause/route.js]: I wonder if this is more confusing than helpful up here

File: thirdeye/thirdeye-frontend/app/pods/rootcause/controller.js [hunk 2]
```diff
@@ -17,6 +18,14 @@ const ROOTCAUSE_SERVICE_BREAKDOWNS = "breakdowns";
 // TODO: Update module import to comply by new Ember standards
 
 export default Ember.Controller.extend({
+  session: Ember.inject.service(),
+
+  /**
+   * User ldap
+   * @type {String}
+   */
+  user: Ember.computed.reads('session.data.authenticated.name'),
```
Comment (apucher) [inline thirdeye/thirdeye-frontend/app/pods/rootcause/controller.js]: I think you can safely read this directly in onSessionSave()

Comment (justYves) [reply thirdeye/thirdeye-frontend/app/pods/rootcause/controller.js]: I believe using onSessionSave would only work for just saved report. I think she's trying to handle the case where it's already saved and we're reading the info.

File: thirdeye/thirdeye-frontend/app/pods/rootcause/route.js [hunk 3]
```diff
@@ -254,6 +260,8 @@ export default Ember.Route.extend(AuthenticatedRouteMixin, {
       sessionId,
       sessionName,
       sessionText,
+      sessionOwner,
```
Comment (apucher) [inline thirdeye/thirdeye-frontend/app/pods/rootcause/route.js]: maybe rename this to "sessionUpdatedBy". The session-owner is the user that originally saved it, which may or may not be the last person to update it.

File: thirdeye/thirdeye-frontend/app/pods/rootcause/controller.js (hunk 2 above)
Comment (ttbach) [reply thirdeye/thirdeye-frontend/app/pods/rootcause/controller.js]: @justYves I think what @apucher means is `user` can be retrieved like `this.get('session.data.authenticated.name` inside the `onSessionSave` method, as opposed to making a CP. I'll make the change.

File: thirdeye/thirdeye-frontend/app/pods/rootcause/route.js (hunk 3 above)
Comment (ttbach) [reply thirdeye/thirdeye-frontend/app/pods/rootcause/route.js]: Gotcha. That makes sense. Will make the change.

File: thirdeye/thirdeye-frontend/app/pods/rootcause/route.js (hunk 1 above)
Comment (ttbach) [reply thirdeye/thirdeye-frontend/app/pods/rootcause/route.js]: I think it's more concise than having to write `model.session.name`, `model.session.text`, etc.

**Pull request merged**